shiny/ui/_input_text.py

```diff
@@ -163,7 +163,10 @@ def input_text_area(
     if autoresize:
         classes.append("textarea-autoresize")
         if rows is None:
-            rows = 1
+            # Use a placeholder of `_auto_` (an invalid value) to signal that the textarea
+            # should autoresized on load
+            # rows = "_auto_"
+            rows = 0
```

Comment:
If we do this, I think we'll want to make the same change to `chat.ts` 

(Although, ideally, I'd like to avoid using `rows` as a signal for resize-on-visible)

Reply:
It has been removed. It will still auto resize once visible. So I don't know where chat.ts will need to be updated